src/pudl/transform/eia861.py

```diff
@@ -692,63 +712,57 @@ def _clean_nerc_add_rows(df, idx_cols, idx_no_nerc):
         df (pandas.DataFrame): A DataFrame with the column 'nerc_region' to be
             cleaned.
         idx_cols (list): A list of the primary keys.
-        idx_no_nerc (list): A list of the primary keys, not including nerc_region.
 
     Returns:
         pandas.DataFrame: A DataFrame with correct and clean nerc regions.
 
     """
+    idx_no_nerc = idx_cols.copy()
+    if 'nerc_region' in idx_no_nerc:
+        idx_no_nerc.remove('nerc_region')
+
     # Split raw df into primary keys plus nerc region and other value cols
     nerc_df = df[idx_cols]
     other_df = df.drop('nerc_region', axis=1).set_index(idx_no_nerc)
 
+    # Backfill the utility with strange numeric nerc region
+    nerc_df.loc[nerc_df.utility_id_eia == 55959, 'nerc_region'] = 'MRO'
+
     # Make all values upper-case
     # Replace all NA values with UNK
-    # Make nerc values into lists to see how many separate values are stuffed into one row (ex: SPP & ERCOT)
+    # Make nerc values into lists to see how many separate values are stuffed into one row (ex: 'SPP & ERCOT' --> ['SPP', 'ERCOT'])
     nerc_df = (
         nerc_df.assign(
             nerc_region=(lambda x: (
                 x.nerc_region
                 .str.upper()
                 .fillna('UNK')
-                .str.findall(r'[A-Z]+'))),
-            multiple_nercs=(lambda x: (
-                x.nerc_region.apply(lambda x: len(x) > 1)))
+                .str.findall(r'[A-Z]+')))
         )
     )
 
-    # Needed nerc_regions as a list to calculate multiple_nercs, but now can convert nerc lists into
-    # comma-separated strings (can't use same col name twice in one assign)
-    nerc_df['nerc_region'] = nerc_df['nerc_region'].str.join(',')
-
-    # Make dfs of multiple vs. single nerc values
-    nerc_multiples = (
-        nerc_df[nerc_df['multiple_nercs']].copy()
-        .drop('multiple_nercs', axis=1)
-    )
-    nerc_singles = (
-        nerc_df[~nerc_df['multiple_nercs']].copy()
-        .drop('multiple_nercs', axis=1)
-    )
-
-    new_nerc_rows = (
-        nerc_multiples
-        .set_index(idx_no_nerc)
-        .stack()
-        .str.split(',', expand=True)
-        .stack()
-        .unstack(-2)
-        .reset_index(-1, drop=True)
-        .reset_index()
-    )
+    # Record a list of the reported nerc regions not included in the recognized regions list (these eventually become UNK)
+    nerc_col = nerc_df['nerc_region'].tolist()
+    nerc_list = list(set([item for sublist in nerc_col for item in sublist]))
+    non_nerc_list = [
+        nerc_entity for nerc_entity in nerc_list if nerc_entity not in RECOGNIZED_NERC_REGIONS + list(NERC_SPELLCHECK.keys())]
+    logger.info(
+        f'The following reported NERC regions are not currently recognized and become UNK values: {non_nerc_list}')
```

Comment:
I really like what you are doing here! I know you said it feels clunky but I actually think it is very clear for the problem you are trying to solve.